lib/internal/fs/watchers.js

```diff
@@ -174,6 +174,12 @@ FSWatcher.prototype[kFSWatchStart] = function(filename,
   }
 };
 
+// To maximize backword-compatiblity for the end user
```

Comment:
```suggestion
// To maximize backward-compatiblity for the end user
```

Reply:
@Trott thanks.  as native english speaker, English is Hard :)